.github/ISSUE_TEMPLATE/internal_story.yml

```diff
@@ -0,0 +1,58 @@
+name: (Internal) Feature Story Template
+description: Intended to help with a template for breaking down larger efforts
+labels: ["priority/high"]
+body:
+  - type: input
+    id: goal
+    attributes:
+      label: Goal
+      description: A high level goal for this feature story.
+      placeholder: Implements the table for the global <feature> page.
+    validations:
+      required: true
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependency issue
+      description: |
+        A link to the story/stories that precede this ticket
+
+        Tip: Using a bullet list will help display links to other tickets by unraveling the name and status of that ticket.
+      placeholder: |
+        * #xxxx1
+        * #xxxx2
+      value: No dependencies
+    validations:
+      required: false
+  - type: textarea
+    id: describe-feature
```

Comment:
```suggestion
    id: describe-feature
```

Reply:
What am I missing? 
![image](https://user-images.githubusercontent.com/8126518/232594056-0377405c-815f-4831-b396-25c76f55e6fd.png)

they are all the same...

Reply:
sorry, i am not sure why i thought that was a typo.